dosing_pump_controller/DisplayController.cpp

```diff
@@ -0,0 +1,70 @@
+//  Created by Andrew Florjancic on 9/19/24.
+//  Copyright © 2024 Andrew Florjancic. All rights reserved
+#include "DisplayController.h"
+
+DisplayController::DisplayController(Display& display, StorageManager& storage_manager)
+: display(display), storage_manager(storage_manager) { 
+    brightness_menu = DoublyLinkedList<uint8_t>();
+    contrast_menu = DoublyLinkedList<uint8_t>();
+    // Append percentages 0-100 in increments of 5
+    for(uint8_t i = 0; i <= 20; i++) {
+        brightness_menu.append(i*5);
+        contrast_menu.append(i*5);
+    }
+    // Set the current brightness and contrast menu items to the values from device storage
+    brightness_menu.setCurrentItem(storage_manager.getBrightness());
+    contrast_menu.setCurrentItem(storage_manager.getContrast());
+}
+
+void DisplayController::setLCDBrightness() {
+    // Convert percentage value to uint8_t value
+    uint8_t brightness = ((double)brightness_menu.getCurrentItem()->data / 100.00) * 255.00; 
```

Comment:
Ideally this calculation should be part of some math library with a `percentageToByte()` function or something. We can worry about that later.